popoolation2.rb

```diff
@@ -0,0 +1,101 @@
+  depends_on "r" => [:run, :optional]
+  depends_on "samtools" => [:run, :optional]
+  depends_on "bwa" => [:run, :optional]
+
+  def install
+    rm "mpileup2sync.pl"
+    chmod 0755, Dir.glob("*.pl")
+    chmod 0755, Dir.glob("export/*.pl")
+    chmod 0755, Dir.glob("indel_filtering/*.pl")
+    mkdir "foo"
+    mv Dir.glob("*.pl"), "foo/"
+    mv Dir.glob("*.jar"), "foo/"
+    mkdir "foo/export"
+    mv Dir["export/*.pl"], "foo/export/"
+    mkdir "foo/indel_filtering"
+    mv Dir["indel_filtering/*.pl"], "foo/indel_filtering/"
+    mkdir "foo/Modules"
+    mv Dir["Modules/*.pm"], "foo/Modules/"
+    mkdir "foo/Modules/Test"
+    mv Dir["Modules/Test/*.pm"], "foo/Modules/Test/"
+    mkdir "bar"
+    system "cpanm", "-L", "bar", "Text::NSP::Measures::2D::Fisher::twotailed"
+    mv "bar/lib/perl5/Text", "foo/Modules/"
+    File.write("ppool2", <<-EOS.undent)
+      \#!/bin/bash
+      atom="$1"
+      if [ "x$atom" != "x" ]; then
+        shift
+        if [ "$atom" = "mpileup2sync" ]; then
+          exec java ${JAVA_OPTS} -jar #{pkgshare}/mpileup2sync.jar "$@"
+        else
+          case "${atom}" in
+            "cmh-test"|"create-genewise-sync"|"fisher-test"|"fst-sliding"|"mpileup2sync"|\\
+            "snp-frequency-diff"|"subsample-synchronized"|"synchronize-pileup"|\\
+            "export/cmh2gwas"|"export/pwc2igv"|"export/subsample_sync2GenePop"|"export/subsample_sync2fasta"|\\
+            "indel_filtering/filter-sync-by-gtf"|"indel_filtering/identify-indel-regions")
+              script=#{pkgshare}/${atom}.pl
+              exec ${script} "$@"
+              ;;
+          esac
+        fi
+      fi
+      echo "Usage: $0 <command> <arguments>
+
+            where <command> is one of
+
+              help
+              cmh-test
+              create-genewise-sync
+              fisher-test
+              fst-sliding
+              mpileup2sync
+              snp-frequency-diff
+              subsample-synchronized
+              synchronize-pileup
+              export/cmh2gwas
+              export/pwc2igv
+              export/subsample_sync2GenePop
+              export/subsample_sync2fasta
+              indel_filtering/filter-sync-by-gtf
+              indel_filtering/identify-indel-regions
+
+           See https://sourceforge.net/p/popoolation2/wiki/Main/ for details."
+    EOS
+    bar=share/"popoolation2"
+    bar.install Dir["foo/*"]
+    bin.install "ppool2"
+  end
+
+  def caveats
+    <<-EOS.undent
+      Please cite as:
+
+        Kofler R, Vinay Pandey R, Schloetterer C. PoPoolation2: Identifying differentiation between
+        populations using sequencing of pooled DNA samples (Pool-Seq). Bioinformatics 27, 3435–3436 (2011)
+        https://doi.org/10.1093/bioinformatics/btr589
+
+      You may also be interested in our Pool-seq review (Nature Reviews Genetics) where we provide some
+      recommendations for the analysis of Pool-seq data: https://doi.org/10.1038/nrg3803
+    EOS
```

Comment:
This is not a caveat. I'd remove the review text and add the citation to the description (maybe to the launch-script help). Here I would add the following caveat (using the names from the rest of the review):

```
The mpileup2sync command is using the java version, which is faster.
Pass java options via the environment variable JAVA_OPTS for that command.

The PoPoolation2 files are installed to
        #{HOMEBREW_PREFIX}/share/.popoolation2
And they are run by a custom launch-script (popoolation2).
```

Reply:
done